configuration/pih/reports/sql/covid19encounterexport.sql

```diff
@@ -6,98 +6,1328 @@
+                    AND value_coded = CONCEPT_FROM_MAPPING('CIEL', '845')
+                    AND encounter_id IN (SELECT 
+                        encounter_id
+                    FROM
+                        encounter
+                    WHERE
+                        encounter_type = @covid_admission_encounter_type))
+            AND concept_id = CONCEPT_FROM_MAPPING('PIH', 'Radiology report comments')
+            AND voided = 0) o1 ON e.patient_id = o1.person_id
+        AND e.encounter_id = o1.encounter_id 
+SET 
+    abUS_findings = o1.value_text;
+
+-- Other finding
+UPDATE temp_encounter e
+        LEFT JOIN
+    (SELECT 
+        encounter_id, person_id, obs_group_id, value_coded
+    FROM
+        obs
+    WHERE
+        obs_group_id IN (SELECT 
+                obs_group_id
+            FROM
+                obs o
+            WHERE
+                voided = 0
+                    AND concept_id = CONCEPT_FROM_MAPPING('PIH', 'Radiology procedure performed')
+                    AND value_coded NOT IN (CONCEPT_FROM_MAPPING('CIEL', '165152') , CONCEPT_FROM_MAPPING('CIEL', '163041'), CONCEPT_FROM_MAPPING('CIEL', '845'))
+                    AND encounter_id IN (SELECT 
+                        encounter_id
+                    FROM
+                        encounter
+                    WHERE
+                        encounter_type = @covid_admission_encounter_type))
+            AND concept_id = CONCEPT_FROM_MAPPING('PIH', 'Radiology procedure performed')
+            AND voided = 0) o1 ON e.patient_id = o1.person_id
+        AND e.encounter_id = o1.encounter_id 
+SET 
+    radiology_other = CONCEPT_NAME(o1.value_coded, 'fr');
+
+UPDATE temp_encounter e
+        LEFT JOIN
+    (SELECT 
+        encounter_id, person_id, obs_group_id, value_text
+    FROM
+        obs
+    WHERE
+        obs_group_id IN (SELECT 
+                obs_group_id
+            FROM
+                obs o
+            WHERE
+                voided = 0
+                    AND concept_id = CONCEPT_FROM_MAPPING('PIH', 'Radiology procedure performed')
+                    AND value_coded NOT IN (CONCEPT_FROM_MAPPING('CIEL', '165152') , CONCEPT_FROM_MAPPING('CIEL', '163041'), CONCEPT_FROM_MAPPING('CIEL', '845'))
+                    AND encounter_id IN (SELECT 
+                        encounter_id
+                    FROM
+                        encounter
+                    WHERE
+                        encounter_type = @covid_admission_encounter_type))
+            AND concept_id = CONCEPT_FROM_MAPPING('PIH', 'Radiology report comments')
+            AND voided = 0) o1 ON e.patient_id = o1.person_id
+        AND e.encounter_id = o1.encounter_id 
+SET 
+    radiology_other_comments = o1.value_text;
+
+UPDATE temp_encounter 
+SET 
+    disposition = OBS_VALUE_CODED_LIST(encounter_id,
+            'PIH',
+            'Hum Disposition categories',
+            'fr');
+UPDATE temp_encounter 
+SET 
+    admission_ward = (SELECT 
```

Comment:
If we don't already have a function to get a location_id from a lookup (name, uuid, metadata mapping), then it should be added and utilized here.